### PR TITLE
Release notes for Percona Server 5.5.48-37.8 (5.5)

### PR DESCRIPTION
**Pull request by hrvojem**
Fixes LP bugs Bug #1349085 and Bug #1516524 by adding missing bugs to
the "upstream bugs fixed" list

### CODE REVIEW

File: doc/source/release-notes/Percona-Server-5.5.48-37.8.rst [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+Bugs Fixed
+==========
+
+ With :ref:`expanded_innodb_fast_index_creation` enabled, DDL queries involving |InnoDB| temporary tables would cause later queries on the same tables to produce warnings that their indexes were not found in the index translation table. Bug fixed :bug:`1233431`.
+
+ Package upgrade on *Ubuntu* would run ``mysql_install_db`` even though data directory already existed. Bug fixed :bug:`1457614`.
+
+ Starting |MySQL| with ``systemctl`` would fail with timeout if socket was specified with custom path. Bug fixed :bug:`1534825`.
+
+ Because ``mysqldumpslow`` script was not compatible with |Percona Server| extended slow query log format it wasn't working correctly. Please use `pt-query-digest <https://www.percona.com/doc/percona-toolkit/2.2/pt-query-digest.html>`_ from |Percona Toolkit| instead. Bug fixed :bug:`856910`.
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.5.48-37.8.rst]: Doesn't say what was actually done in this release

File: doc/source/release-notes/Percona-Server-5.5.48-37.8.rst [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+Based on `MySQL 5.5.48 <http://dev.mysql.com/doc/relnotes/mysql/5.5/en/news-5-5-48.html>`_, including all the bug fixes in it, |Percona Server| 5.5.48-37.8 is now the current stable release in the 5.5 series. All of |Percona|'s software is open-source and free, all the details of the release can be found in the `5.5.48-37.8 milestone at Launchpad <https://launchpad.net/percona-server/+milestone/5.5.48-37.8>`_. 
+
+Bugs Fixed
+==========
+
+ With :ref:`expanded_innodb_fast_index_creation` enabled, DDL queries involving |InnoDB| temporary tables would cause later queries on the same tables to produce warnings that their indexes were not found in the index translation table. Bug fixed :bug:`1233431`.
+
+ Package upgrade on *Ubuntu* would run ``mysql_install_db`` even though data directory already existed. Bug fixed :bug:`1457614`.
+
+ Starting |MySQL| with ``systemctl`` would fail with timeout if socket was specified with custom path. Bug fixed :bug:`1534825`.
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.5.48-37.8.rst]: the socket with a custom path?

**Comment by hrvojem** [comment]
Fixed

**Comment by laurynas-biveinis** [comment]
LGTM

**Pull request merged**